blockly-core/core/ui/block.js

```diff
@@ -319,9 +319,6 @@ Blockly.Block.terminateDrag_ = function() {
   if (selected) {
     selected.blockSpace.fireChangeEvent();
     selected.blockSpace.blockSpaceEditor.setCursor(Blockly.Css.Cursor.OPEN);
-  } else {
-    // If not, at least trigger a cursor change on blocks.
-    Blockly.Css.setCursor(Blockly.Css.Cursor.OPEN, null);
```

Comment:
Brad added it here: https://github.com/code-dot-org/code-dot-org/commit/dbd61e619f4a353b30692be03461bcb575f15bd8#diff-a6d3fad5bc46577cdf376f3a91c20361R287

He suggested testing block trashing in the regular trashcan, function editor, and mutator flyouts.  I've checked these scenarios and didn't find any regressions, but I could still be missing an edge case.